src/httprpc.cpp

```diff
@@ -226,6 +226,10 @@ static bool InitRPCAuthentication()
         LogPrintf("Config options rpcuser and rpcpassword will soon be deprecated. Locally-run instances may remove rpcuser to use cookie-based auth, or may be replaced with rpcauth. Please see share/rpcuser for rpcauth auth generation.\n");
         strRPCUserColonPass = gArgs.GetArg("-rpcuser", "") + ":" + gArgs.GetArg("-rpcpassword", "");
     }
+    if (gArgs.GetArg("-rpcauth","") != "")
+    {
+        LogPrintf("RPCAuth user has been found.\n");
```

Comment:
I don't think this log message 'user has been found' is very illuminating, out of the blue. Something like `Using rpcauth authentication` would be clearer, I think.

Reply:
I 100% agree with you and had that originally. The reason I changed it is because I do no verification that the entries are valid, just that they exist. I wanted to put the log wherever the client did the verification of them; but, as far as I can tell, they aren't even looked at until an RPC user authorization is attempted. I didn't want to promise that they are being used without doing so, and I didn't want to add any overhead in checking them without talking about it here first. I'd be happy to change it if you would like.